articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Multi-factor Authentication in Auth0
+description: The basics of multi-factor authentication and the different methods of implementing it with Auth0.
+url: /multifactor-authentication
+toc: true
+topics:
+    - mfa
+contentType:
+  - index
+useCase:
+  - customize-mfa
+---
+# Multi-factor Authentication
+
+## What is multi-factor authentication?
+
+Multi-factor Authentication (MFA) is a method of verifying a user's identity by requiring them to present more than one piece of identifying information. This method provides an additional layer of security, decreasing the likelihood of unauthorized access. The type of information required from the user is typically two or more of the following:
+
+* **Knowledge**: Something the user **knows** (such as a password)
+* **Possession**: Something the user **has** (such as a mobile device)
+* **Inheritance**: Something the user **is** (such as a fingerprint or retina scan)
+
+## Implement MFA with Auth0
+
+Enabling MFA for your tenant is a fairly straightforward process. First, you toggle on the factors you choose to enable on your tenant, such as push notifications or SMS. Next, you perform any further setup required to configure that factor, and last, you choose whether you wish to force MFA for all users or not. See the instructions below for details.
+
+You can also [customize your MFA flow](/multifactor-authentication/custom) with [Auth0 Rules](/rules), to allow MFA to only be required in specific circumstances or force a particular factor to be used.
+
+### 1. Toggle on the factors you require in the Dashboard
```

Comment:
Minor: I read this as 'factors I require in the dashboard' instead of toggling in the dashboard the factors I require :).